core/client-internal/src/main/java/tachyon/client/file/policy/FileWriteLocationPolicy.java

```diff
@@ -46,6 +46,6 @@
    * @param blockSizeBytes the size of the block in bytes
    * @return the address of the worker to write to
    */
-  public NetAddress getWorkerForNextBlock(List<BlockWorkerInfo> workerInfoList,
+  public WorkerNetAddress getWorkerForNextBlock(List<BlockWorkerInfo> workerInfoList,
```

Comment:
Since this class is public API, ideally this change can be included in this release. Is it possible to separate adding wire structure out from this PR?